src/predict.rs

```diff
@@ -146,6 +145,15 @@ impl PredictionMode {
 
     let variant = PredictionVariant::new(x, y);
 
+    let alpha = match intra_param {
+      IntraParam::Alpha(val) => val,
+      _ => 0,
+    };
+    let angle_delta = match intra_param {
+      IntraParam::Angle_delta(val) => val,
+      _ => 0,
+    };
+
```

Comment:
These matches could be merged with the match at line 168. It may not be obvious how to cover all cases correctly, so we can take care of this in a future pull request if necessary.

Reply:
Yes. It will be improved in future PR.